onetl/connection/db_connection/kafka/kafka_ssl_protocol.py

```diff
@@ -0,0 +1,134 @@
+#  Copyright 2023 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+
+from pydantic import Field, SecretStr, validator
+
+from onetl._util.file import is_file_readable
+from onetl.impl import GenericOptions, LocalPath
+
+if TYPE_CHECKING:
+    from onetl.connection import Kafka
+
+from onetl.connection.db_connection.kafka.kafka_protocol import KafkaProtocol
+
+
+class KafkaSSLProtocol(KafkaProtocol, GenericOptions):
+    """
+    Class encapsulates all the necessary configurations and interactions for managing
+    SSL protocols within a Kafka system. It serves as a handler for different keystore types and truststore configurations,
+    allowing the connection to Kafka brokers through SSL.
+
+    See:
+    * `IBM Documentation <https://www.ibm.com/docs/en/cloud-paks/cp-biz-automation/19.0.x?topic=fcee-kafka-using-ssl-kerberos-authentication>`_
+    * `Kafka Documentation <https://kafka.apache.org/090/documentation.html>`_
+    * `How to use PEM Certificates with Kafka <https://codingharbour.com/apache-kafka/using-pem-certificates-with-apache-kafka/>`_
+
+    Examples
+    --------
+
+    PEM certificates as raw strings:
+
+    .. code:: python
+
+        ssl = (
+            Kafka.SSLProtocol(
+                keystore_type="PEM",
+                keystore_certificate_chain="-----BEGIN CERTIFICATE-----MIIDZjC...-----END CERTIFICATE-----",
+                keystore_key="-----BEGIN ENCRYPTED PRIVATE KEY-----MIIDZjC..-----END ENCRYPTED PRIVATE KEY-----",
+                key_password="<private_key_password>",
```

Comment:
```suggestion
                key_password="password",
```
Мало ли какой-нибудь пользователь поймет это как "пароль должен быть внутри знаков `<>`